docs/content/querying/filters.md

```diff
@@ -80,3 +80,36 @@ The following matches any dimension values for the dimension `name` between `'ba
   "function" : "function(x) { return(x >= 'bar' && x <= 'foo') }"
 }
 ```
+
+### Extraction filter
+
+Extraction filter matches a dimension using some specific [Extraction function](./dimensionspecs.html#extraction-functions).
+The filter matches the values for which the extraction function has transformation entry `input_key=output_value` where
+ `output_value` is equal to the filter `value` and `input_key` is present as dimension.
+
+**Example**
+The following matches dimension values in `[product_1, product_3, product_5]` for the column `product`
```

Comment:
Leaving the article out makes it awkward English. If you wish to add specificity by adding the word "filter," then still it should be "The following filter matches ...."